WalletWasabi.Gui/MainWindow.xaml

```diff
@@ -11,7 +11,7 @@
                       xmlns:idc="clr-namespace:Dock.Avalonia.Controls;assembly=Dock.Avalonia"
                       xmlns:beh="clr-namespace:AvalonStudio.Utils.Behaviors;assembly=AvalonStudio.Utils"
                       xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-                      Icon="avares://WalletWasabi.Gui/Assets.WasabiLogo256.png"
+                      Icon="avares://WalletWasabi.Gui/Assets/WasabiLogo256.png"
```

Comment:
@lontivero In your PR https://github.com/zkSNACKs/WalletWasabi/pull/1367 where you put the images to another place this would've caused issues for example here.